operational/azure/total_instance_usage_report/README.md

```diff
@@ -0,0 +1,54 @@
+# Azure Usage Report - Instance Time Used
+
+## What It Does
+
+This Policy Template leverages Flexera CCO APIs to produce a stacked bar chart showing Total Instance Time for Azure Virtaul Machine Families used per month for the last 12 months. The user can specify which unit of time and which instance spec to use to normalize time against; for example, a chart can be produced for total instance vCPU hours. The data feeding this chart can be filtered by Azure region or Flexera Billing Center if desired. Optionally, the result can be emailed.
+
+## How It Works
+
+- This policy supports a single Azure region or the entire Organization.
+- This policy produces a stacked-bar chart showing Total Instance Units by Virtual Machine Family for the top 8 most used Virtual Machine Families. All other Virtual Machine Families will be aggregated and displayed as "Other". Values shown in the graph are for the past 12 months.
+- The unit used for normalizing instance usage depends on the value of the `Instance Unit` parameter.
+- The unit used for time depends on the value of the `Time Unit` parameter.
+
+## Input Parameters
+
+- *Email Addresses* - A list of email addresses to notify.
+- *Months Back* - How many months to go back in the chart/incident.
+- *Instance Unit* - Instance (Virtual Machine) unit to normalize usage against. Supported units:
+  - `Normalized Units (NFUs)` - Normalize the data against each [Instance Size Flexibility (ISF) Ratio](https://learn.microsoft.com/en-us/azure/virtual-machines/reserved-vm-instance-size-flexibility) of each Virtual Machine instance.
```

Comment:
This parameter value (and the README) should be renamed from NFUs (which is AWS specific) to something more reflective of the Azure platform.